tasks/generate-sitemap.mjs

```diff
@@ -1,32 +1,13 @@
 // Adapted from https://gist.github.com/joranquinten/78f3e288274a3c9405a499b8a8c46e35
 import * as fs from 'fs';
 import { execSync } from 'child_process';
-import { createRequire } from 'module';
-import { directory } from '../src/directory/directory.mjs';
-
-const require = createRequire(import.meta.url);
-const generatePathMap = require('../generatePathMap.cjs');
+import directory from '../src/directory/directory.json' assert { type: 'json' };
```

Comment:
importing the newly generated directory json

Reply:
The `assert` addition is cool, is it recommended to do that when we import a JSON file to make sure it's actually json basically?